### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by rhgndf** [PhantomSDR/PhantomSDR#18] [Announcements]
Windows binary.

Note that the html path is different for this package

<hr /><em>This discussion was created from the release <a href='https://github.com/PhantomSDR/PhantomSDR/releases/tag/'>v0.1.0</a>.</em>

**Comment by fhcastellani** [comment]
Please, where the **windows binaries** can be found ? Thanks in advance !

**Reply by rhgndf** [reply]
Here: https://github.com/PhantomSDR/PhantomSDR/releases/tag/v0.1.0

**Reply by fhcastellani** [reply]
Thank you ! I will try my best here to make it work!

**Reply by fhcastellani** [reply]
Is there any Windows Binary for the rx888_stream? I think it is supposed to start the streaming and then start the server... Sorry to ask, but I think I need some tutorial "for dummies" in my case... Thanks again!

**Reply by fhcastellani** [reply]
> Here: https://github.com/PhantomSDR/PhantomSDR/releases/tag/v0.1.0

Got the server running on windows 11. A friend compiled the rx888_stream (the original C code, not RUST code) to win binary. RX888 connecter to spectrumserver and webpage loaded, it was online on the servers list, BUT with no audio or watterfall.

Any advice is welcome to make it work and put it online.

Thank you !
![IMG-20240525-WA0001](https://github.com/PhantomSDR/PhantomSDR/assets/170742633/b021f029-5b43-4c8b-84f9-f1c718358564)
![IMG-20240525-WA0003](https://github.com/PhantomSDR/PhantomSDR/assets/170742633/48e09f10-f772-4e14-a995-d39a50b0b6eb)

**Reply by rhgndf** [reply]
That is weird, I have used that binary to deploy to a windows machine without issues.

Try opening the developer console with F12 and see if there are any errors.

**Reply by fhcastellani** [reply]
I do suspect that the problem is with my **RX888_stream windows binary**... it is not accepting the --pga -o options... 
Attached is the console response to the command when I run the rx888_stream and them connect to the server from the browser on **localhost:9002**.
I attached my version of the RX888_stream, so you can see if it works (I do suspect the problem is there...).
I would kindly ask if you can post your windows binary of the RX888_stream, or, if this can be included in the package of the Windows Binaries of PhantomServer, so I can try to run again.
Thanks !
![image](https://github.com/PhantomSDR/PhantomSDR/assets/170742633/9afe920e-38d0-47d4-a093-7da0906a1168)
[RX888_stream_Win.zip](https://github.com/PhantomSDR/PhantomSDR/files/15445584/RX888_stream_Win.zip)
